src/components/shared/Backtrace.css

```diff
@@ -1,49 +1,23 @@
 .backtrace {
-  --gradient-height: calc(var(--max-height) - 5em);
-
-  /* Box */
-  max-width: 35em;
+  max-width: var(--tooltip-detail-max-width);
   padding: 0;
-  margin: 5px;
-
-  /* Other */
-  font-size: 0.9em;
-  line-height: 1.5;
+  margin: 0;
 }
 
 .tooltip .backtrace {
   overflow: hidden;
-  max-height: var(--max-height);
-
-  /* The gradient is defined towards the bottom so that we hide the end of the
-   * backtrace only when the element is tall enough, which is when the backtrace
-   * is long. */
-
-  -webkit-mask-image: linear-gradient(
-    to bottom,
-    white,
-    white var(--gradient-height),
-    transparent
-  );
-  mask-image: linear-gradient(
-    to bottom,
-    white,
-    white var(--gradient-height),
-    transparent
-  );
 }
 
 .backtraceStackFrame {
   /* Position */
-  display: block;
   overflow: hidden;
 
   /* Box */
   padding: 0;
   margin: 0;
+  list-style-position: inside;
```

Comment:
This is a bit unpretty on my computer (zoom is 100% so it's not even because of zoom):  
![image](https://user-images.githubusercontent.com/454175/92948835-4e882280-f45a-11ea-88c3-ffedd8fc7866.png)
I'd rather keep it at "none" for now...

Reply:
I made it "none", but added a horizontal rule to differentiate it.

<img width="923" alt="image" src="https://user-images.githubusercontent.com/1588648/93241133-8e952100-f74a-11ea-9b16-5169aba37052.png">